data/modules/StationTrafficControl/StationTrafficControl.lua

```diff
@@ -106,8 +109,12 @@ end
 local loaded_data
 
 local onGameStart = function ()
+	local station = Game.player:FindNearestTo("SPACESTATION")
 	if (loaded_data) then
 		playerInControlledSpace = loaded_data.playerInControlledSpace
+	elseif Game.player.flightState == "DOCKED" or
+			station and station:DistanceTo(Game.player) < station.lawEnforcedRange then
```

Comment:
@impaktor I added **'station and'** here in response to your earlier comment on station being nil. I use no parenthesis here and trust in short-circuit evaluation. This is OK and clean right?